resources/assets/js/components/JobDetails/JobDetails.tsx

```diff
@@ -950,7 +955,10 @@ const JobDetails: React.FunctionComponent<
                         province={intl.formatMessage(
                           provinceName(Number(values.province)),
                         )}
+                        education={values.education}
```

Comment:
Education should come from the Job object here

Reply:
Makes sense 👀 Done 👍